deploy/examples/object-multisite.yaml

```diff
@@ -36,6 +36,9 @@ spec:
       requireSafeReplicaSize: true
     parameters:
       compression_mode: none
+  #customEndpoints:
+  #  - "http://rgw-a.fqdn"
```

Comment:
the documentations says a port is mandatory

Reply:
Port is not mandatory, for example, the openshift-route's(Ingress load balancer used in openshift) endpoint does not usually have the port.  I will update the doc to reflect the same